src/test/java/org/mockitousage/debugging/InvocationsPrinterTest.java

```diff
@@ -0,0 +1,81 @@
+package org.mockitousage.debugging;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.internal.debugging.InvocationsPrinter;
+import org.mockitousage.IMethods;
+import org.mockitoutil.TestBase;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+public class InvocationsPrinterTest extends TestBase {
+
+    @Mock IMethods mock;
+
+    @Test public void no_invocations() {
+        Assert.assertEquals("No interactions and stubbings found for mock: mock", new InvocationsPrinter().printInvocations(mock));
```

Comment:
It would be nice to prefer AssertJ assertions over Junit they have a better API and allow chaining. Not a blocker to merge though.